x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/authc/service/ServiceAccountToken.java

```diff
@@ -44,10 +45,6 @@ public String getTokenName() {
         return tokenName;
     }
 
-    public String getQualifiedName() {
-        return getAccountId().asPrincipal() + '/' + tokenName;
-    }
```

Comment:
You may want this back when you introduce the file credential store - you need some way to key the file - but we can work that out then.

Reply:
Yes I deleted it because it not used and shows up gray in IDE and this somehow annoyed me. Will restore it later.